### ISSUE TITLE
Question about workflow

### ISSUE THREAD

**Issue by yarkable** [open-mmlab/mmdetection#4838]
May I ask that, by default, we get workflow [('train', 1)] and evaluate per epoch, is it the same as [('train', 1), ('val', 1)] ? I really don't know the difference

**Comment by ZwwWayne** [comment]
They are different, `[('train', 1), ('val', 1)]` will calculate the loss of validation set, which might be beneficial for analysis.
